.github/workflows/argilla.yml

```
@@ -53,6 +53,10 @@ jobs:
       - name: Wait for argilla-quickstart to start
         run: |
           while ! curl -XGET http://localhost:6900/api/_status; do sleep 5; done
+      - name: Set huggingface hub credentials
+        run: |
+          echo "HF_TOKEN_ARGILLA_INTERNAL_TESTING=${{ secrets.HF_TOKEN_ARGILLA_INTERNAL_TESTING }}" >> "$GITHUB_ENV"
```

IF the [`HF_TOKEN`](https://huggingface.co/docs/huggingface_hub/en/package_reference/environment_variables#hftoken) is the standard environment variable name, what if we create it based on the CI secrets?
```suggestion
          echo "HF_TOKEN=${{ secrets.HF_TOKEN_ARGILLA_INTERNAL_TESTING }}" >> "$GITHUB_ENV"
```